src/protocol/message.rs

```diff
@@ -172,6 +172,8 @@ pub enum Message {
     Pong(Vec<u8>),
     /// A close message with the optional close frame.
     Close(Option<CloseFrame<'static>>),
+    /// Raw frame
```

Comment:
```suggestion
    /// Raw frame. Note, that you're not going to get this value while reading the message.
    /// This is meant for the sending part in case sending of the raw frames is desired.
    /// See the [discussion](https://github.com/snapview/tungstenite-rs/pull/250) for more details.
```

Reply:
Thanks. Done.